src/configs/recommended.ts

```diff
@@ -56,5 +57,14 @@ export function recommended(extenions?: RuleExtensions): FlatConfig.ConfigArray
         ...tsRules(extenions?.ts),
       },
     },
+    {
+      plugins: {
+        import: pluginImport,
```

Comment:
I think we only want to add this plugin for TS and JS files?

Reply:
If we limit this to just TS and JS by using `files: [...]` here, projects like TutorialKit cannot leverage this for `*.astro` files. But it looks like there are plenty of rules in this `recommended` config that hard-code the file patterns.

I think we should allow passing custom extensions for all config entries. But that should be done in separate PR. 

I'll add the JS+TS limit here.

Reply:
Yea. I'd say our plugin can re-export the `pluginImport` and the rules so you can add that to other extensions as well in your project. But for this eslint-plugin I think it makes sense to limit it to JS/TS by default.